packages/destination-actions/src/destinations/gainsight-px-cloud-action/index.ts

```diff
@@ -0,0 +1,104 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import { IntegrationError } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+import { defaultValues } from '@segment/actions-core'
+
+import { getEndpointByRegion } from './regional-endpoints'
+import identifyUser from './identifyUser'
+import groupIdentify from './groupIdentify'
+import trackEvent from './trackEvent'
+import trackPageView from './trackPageView'
+
+/** used in the quick setup */
+const presets: DestinationDefinition['presets'] = [
+  {
+    name: 'Identify User',
+    subscribe: 'type = "identify"',
+    partnerAction: 'identifyUser',
+    mapping: defaultValues(identifyUser.fields)
+  },
+  {
+    name: 'Group User',
+    subscribe: 'type = "group"',
+    partnerAction: 'groupIdentify',
+    mapping: defaultValues(groupIdentify.fields)
+  },
+  {
+    name: 'Track Event',
+    subscribe: 'type = "track"',
+    partnerAction: 'trackEvent',
+    mapping: defaultValues(trackEvent.fields)
+  },
+  {
+    name: 'Track Page View',
+    subscribe: 'type = "page"',
+    partnerAction: 'trackPageView',
+    mapping: defaultValues(trackPageView.fields)
+  }
+]
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Gainsight Px Cloud Action',
+  slug: 'gainsight-px-cloud-action',
```

Comment:
```suggestion
  slug: 'actions-gainsight-px-cloud',
```
Adjusts slug to fit naming convention